src/utils/schedule-dump.ts

```diff
@@ -0,0 +1,31 @@
+import got from 'got';
+import * as fs from 'fs';
+import { performance } from 'perf_hooks';
+import courses from '../static/courses.json';
+import { Course } from '../types';
+import { forEachHelper } from './common';
+
+const main = async () => {
+  console.log('dump:schedule - starting...\n');
+  const start = performance.now();
+
+  const c: Course[] = courses;
```

Comment:
@VikeLabs/javascript Any idea how to get rid of this TypeScript error?

Reply:
I am not seeing this error? Even in commit `464b96f`

Reply:
Oh I see I don't get the error because I don't have the file that has the courses. Assuming its a type error what you could do is
```suggestion
  const c = courses as Course[];
```

This casts it to the type, which is just you telling typescript "Hey, I know courses is that type". It doesn't actually modify courses in anyway. 

You could do `const myString = 3 as string` which would tell ts that myString is a string, but if you do `typeof myString` later it will be a number still.